alacritty_terminal/CHANGELOG.md

```diff
@@ -0,0 +1,13 @@
+# Changelog
+
+All notable changes to alacritty_terminal are documented in this file. The
+sections should follow the order `Added`, `Changed`, `Deprecated`, `Fixed` and
+`Removed`.
+
+The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
+
+## 0.23.0-dev
+
+### Changed
+
+- `Term` is not focused by default anymore
```

Comment:
I wonder if we should somehow indicate breaking changes? Do you have any opinion on this?

Maybe a migration guide between versions?

Reply:
I'd suggest migration for bigger stuff, you can look at the changelog we have in winit now. For such a small change, I don't think you need it?

https://github.com/rust-windowing/winit/blob/master/src/changelog/unreleased.md

Reply:
> you can look at the changelog we have in winit now

I mean that's exactly the issue right there. A ton of changes but no clear indication which changes require my attention.

I don't care if a bunch of new stuff was added or something was fixed when updating to a new version, but I **do** care if something needs manual intervention like this focus change to keep the behavior consistent.

Some changelogs have ugly prefixes like `[BREAKING]` for these kind of entries, but I don't think that's particularly nice. Maybe we can add a line at the top explaining that breaking changes are bold, then use that to indicate them?

Reply:
>I mean that's exactly the issue right there. A ton of changes but no clear indication which changes require my attention.

It has a big chunk of code and explanation for what you need. So it's just 2 changes where you need attention.

We can do a separate section with breaking changes.

Reply:
> It has a big chunk of code and explanation for what you need. So it's just 2 changes where you need attention.

It doesn't though. It states nowhere that these are the only changes that require my attention. It just tells me what I need to do for those.

Reply:
>It doesn't though. It states nowhere that these are the only changes that require my attention. It just tells me what I need to do for those.

Hm, that's fair, because it'll start to warn you, since we don't remove the API. For real breaking change I just did `bold` iirc.

>Do you suggest adding an extra section with its own Fixed/Added/etc?

Just `Breaking`.

Reply:
I like bold for breaking, it's unintrusive but still pretty clear.